solidity/contracts/Oracle.sol

```diff
@@ -54,9 +59,10 @@ contract Oracle is Ownable {
     currentInternalId += 1;
```

Comment:
It's unrelated to this specific PR, but do you think we should also add something like a timestamp to `RunRequest` so that we can differentiate `currentInternalId` if it wraps on overflow?

Reply:
All event logs, which is what `RunRequest` is come with a `blockNumber` that in many respects is the block chain equivalent of a timestamp. As for `currentInternalId` wrapping on overflow, it is only modified by this line:

```solidity
  function requestData(
    uint256 _version,
    bytes32 _jobId,
    address _callbackAddress,
    bytes4 _callbackFunctionId,
    bytes32 _externalId,
    bytes _data
  )
    public
    onlyLINK
  {
    currentInternalId += 1; // should I use SafeMath just for incrementing? overkill?
```

I want to say we're safe but @rupurt and @se3000, let me know if I'm wrong.

Reply:
That's wrapping after 115792089237316195423570985008687907853269984665640564039457584007913129639935 requests. I think we're safe for now, especially because I think we may have to revisit `currentInternalId` soon for another story.